src/averageAges.js

```diff
@@ -20,6 +20,18 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+  const mens = (century === undefined)
+    ? people.filter(men =>
+      men.sex === 'm').map(men =>
+      men.died - men.born)
+    : people.filter(men =>
+      men.sex === 'm').filter(men =>
+      Math.ceil(men.died / 100) === century).map((men) =>
+      men.died - men.born
+    );
```

Comment:
Couple of comments:

1. Use correct English. Even some typos can make a recruiter prefer other person's work over yours.
2. Result here is an array of ages. More appropriate var name would be smth like `menAges`
3. Use correct indentations when chaining methods.
4. Try to get used to destructuring objects:

```
people
      .filter(({ sex }) => sex === 'm')
      .map(({ died, born }) => died - born)
```

```suggestion
  const menAges = (century === undefined)
    ? people
      .filter(person => person.sex === 'm')
      .map(man => man.died - man.born)
    : people
      .filter(person => person.sex === 'm')
      .filter(man => Math.ceil(man.died / 100) === century)
      .map(man => man.died - man.born);
```